### ISSUE TITLE
Error with latest version of antd

### ISSUE THREAD

**Issue by LikeCarter** [SolidZORO/next-plugin-antd-less#77]
Hi,

It seems we're having trouble compiling the latest version of antd `"4.17.0-alpha.0"`

Is the issue: the less parser does not recognize `@import`?

The error message:
```
/node_modules/antd/lib/button/style/index-default.less)

@import './index';
^
Variable @import is undefined
```

**Comment by LikeCarter** [comment]
Seems it is fixed in `4.17.0-alpha.3`. 

- 🐞 Fix antd compile file miss `/style/default.css`. [#32116](https://github.com/ant-design/ant-design/pull/32116)

**Comment by SolidZORO** [comment]
antd alpha version is very uncertain, especially in next.js, the stable version is recommended.
